CrewChiefV4/Events/OvertakingAidsMonitor.cs

```diff
@@ -62,20 +64,28 @@ override protected void triggerInternal(GameStateData previousGameState, GameSta
                 }
                 if (currentGameState.SessionData.IsNewLap)
                 {
-                    if (drsAvailableOnThisLap && !hasUsedDrsOnThisLap)
+                    if (drsAvailableOnThisLap && !hasUsedDrsOnThisLap
+                        || (CrewChief.gameDefinition.gameEnum == GameEnum.RF2_64BIT && drsAvailableInAnyZoneOnThisLap && !hasUsedDrsOnThisLap))
                     {
                         audioPlayer.playMessage(new QueuedMessage("missed_available_drs", MessageContents(folderDontForgetDRS), 0, this));
                     }
                     drsAvailableOnThisLap = currentGameState.OvertakingAids.DrsAvailable;
+                    drsAvailableInAnyZoneOnThisLap = false;
                     hasUsedDrsOnThisLap = false;
                     playedGetCloserForDRSOnThisLap = false;
                     playedOpponentHasDRSOnThisLap = false;
                 }
+                if (currentGameState.OvertakingAids.DrsAvailable)
+                {
+                    drsAvailableInAnyZoneOnThisLap = true;
```

Comment:
you just set this to false above

Reply:
this block is necessary because DRS becomes available multiple times during the lap, I believe.